tests/spec/integration_spec.lua

```
@@ -0,0 +1,454 @@
+  schedule = function(fn) fn() end,
+  tbl_extend = function(behavior, ...)
+    local result = {}
+    for _, tbl in ipairs({...}) do
+      for k, v in pairs(tbl) do
+        result[k] = v
+      end
+    end
+    return result
+  end,
+  tbl_deep_extend = function(behavior, ...)
+    return vim.tbl_extend(behavior, ...)
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)